mysite/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for mysite project.
+
+It exposes the ASGI callable as a module-level variable named ``application``.
```

Comment:
I don't think that the framework documentation belongs in docstrings. The generator put it here for your convenience. It's useful for you as a getting started pointer but it's useless in the context of the concrete project as it's about implementing a website, not putting links to django docs everywhere.